to0scheduler/libto0/pom.xml

```diff
@@ -39,10 +39,28 @@
       <groupId>io.projectreactor</groupId>
       <artifactId>reactor-core</artifactId>
     </dependency>
+    
+    <dependency>
+      <groupId>org.mockito</groupId>
```

Comment:
Are these new dependencies or they were being used other IOT-Platform components?

Reply:
Spring-boot test was used earlier. This was added to replace the spring-boot test framework.